config/requirements-dev.txt

```diff
@@ -0,0 +1,5 @@
+# This is for dev purpose
+-r requirements.txt
```

Comment:
cool, I didn't know you could do this.

https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format